axelrod/strategies/grudger.py

```diff
@@ -342,3 +342,75 @@ def strategy(opponent: Player) -> Action:
         elif opponent.defections:
             return D
         return C
+
+
+class Capri(Player):
+    """
+    CAPRI is a memory-3 strategy proposed in [Murase2020]_. Its behavior is defined by the following five rules.
```

Comment:
I've read the paper now (sorry for taking so long!) which I enjoyed, I suggest the (very minor) following clarification re the description:

```suggestion
    CAPRI is a memory-3 strategy proposed in [Murase2020]_. Its behavior is defined in[Murase2020]_ by the following five rules applied to the last 3 moves of the player and the opponent:
```

I realise this is implied by the `memory-3` comment.